packages/react-components/react-breadcrumb/src/components/Breadcrumb/Breadcrumb.types.ts

```diff
@@ -1,16 +1,38 @@
 import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
 
+export const BreadcrumbSize = ['small', 'medium', 'large'] as const;
```

Comment:
I know it's a bad practice to export types that's why I tried to do it this way. Because copying types all the time is, also, not good